packages/flutter/test/cupertino/button_test.dart

```diff
@@ -451,6 +453,32 @@ void main() {
     ).decoration as BoxDecoration;
     expect(decoration.color, isSameColorAs(CupertinoColors.systemBlue.darkColor));
   });
+
+  testWidgets('Cupertino button changes mouse cursor when hovered on Web', (WidgetTester tester) async {
```

Comment:
Nit: This could be named similar to your tab bar test. `Hovering over Cupertino button updates cursor to clickable on Web`